src/set_theory/surreal/basic.lean

```diff
@@ -306,5 +323,4 @@ end ordinal
 
 -- We conclude with some ideas for further work on surreals; these would make fun projects.
```

Comment:
Should this become a module-level docstring so that it's visible on the documentation website?

Reply:
Sure, I'll put it at the beginning of the file.